README.md

```diff
@@ -0,0 +1,46 @@
+# Wirepas Mesh API
+
+This repository contains a C implementation of the Wirepas Dual MCU API.
+The dual mcu api allows the exchange of data between the host and the
+Wirepas Mesh sink.
+
+
+## C Library
+
+The C implementation is available from [lib](./lib) and is structured in the
+following folders:
+
+-   **[api](./lib/api)**: header files for the WM dual mcu api
+
+-   **[platform](./lib/platform)**: host platform functions for serial interface handling
```

Comment:
It is more than just serial interface. It contains all the abstraction needed for the platform. Locking mechanism, polling thread, logging, serial and timestamp.